languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction/about.md

```diff
@@ -0,0 +1,84 @@
+### Summary
+
+This exercise showed how to:
```

Comment:
For future reference: the `about.md` documents should not refer to the exercise as they should be consider stand-alone documents.

Reply:
> It looks like this doesn't add symlinks but copies of the concept files, is that correct?

Locally it does show them as symlinks, I think it's just how GitHub renders them?

Could you try checking out the branch locally and seeing if they're actually links?

Reply:
Doesn't look like it:

```
erik@VELPIE:~/v3$ gh pr checkout 2790
Already on 'julia/concept-symlinks'
Your branch is up to date with 'origin/julia/concept-symlinks'.
Already up to date.
erik@VELPIE:~/v3$ cd languages/julia/exercises/concept/
boolean-logic/      encounters/         fibonacci-iterator/
emoji-times/        exercism-matrix/    lasagna/
erik@VELPIE:~/v3$ cd languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction/
erik@VELPIE:~/v3/languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction$ ls -al
total 16
drwxr-xr-x 2 erik erik 4096 Nov 17 08:59 .
drwxr-xr-x 3 erik erik 4096 Nov 17 08:59 ..
-rw-r--r-- 1 erik erik 2180 Nov 17 08:59 about.md
-rw-r--r-- 1 erik erik  123 Nov 17 08:59 links.json
erik@VELPIE:~/v3/languages/julia/exercises/concept/lasagna/.meta/concepts/functions-introduction$ cd ..
erik@VELPIE:~/v3/languages/julia/exercises/concept/lasagna/.meta/concepts$ ls -al
total 12
drwxr-xr-x 3 erik erik 4096 Nov 17 08:59 .
drwxr-xr-x 3 erik erik 4096 Nov 17 08:59 ..
drwxr-xr-x 2 erik erik 4096 Nov 17 08:59 functions-introduction
erik@VELPIE:~/v3/languages/julia/exercises/concept/lasagna/.meta/concepts$ cd ..
erik@VELPIE:~/v3/languages/julia/exercises/concept/lasagna/.meta$ ls -al
total 24
drwxr-xr-x 3 erik erik 4096 Nov 17 08:59 .
drwxr-xr-x 4 erik erik 4096 Nov  6 12:29 ..
drwxr-xr-x 3 erik erik 4096 Nov 17 08:59 concepts
-rw-r--r-- 1 erik erik  372 Nov  6 12:29 config.json
-rw-r--r-- 1 erik erik  776 Nov  6 12:29 design.md
-rw-r--r-- 1 erik erik  151 Nov  6 12:29 example.jl
```

Reply:
Weird, I'll try again.